cumulus_library_data_metrics/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Data Metrics study for Cumulus Library"""
 
-__version__ = "5.0.1"
+__version__ = "5.1.0"
```

Comment:
This fix itself feels like just a patch bump, but the other unreleased change is actually exporting the meta tables at all, which I had forgotten to do - so a minor bump felt appropriate.